tests/test_export.py

```diff
@@ -4,18 +4,29 @@
 
 # use an intermediate argvalues variable to avoid large pytest backtraces
 _test_reduce_common_paths_arg_values = [
-    (['common/a', 'common/b'],
-     ['common/a', 'common/b', 'old/a', 'old/b'],
-     [],
-     ['common']),
-    (['common/a/x', 'common/a/y', 'common/b/x', 'common/b/y'],
-     ['common/a/x', 'common/a/y', 'common/b/x', 'common/b/y'],
-     ['common/a/z'],
-     ['common/a/x', 'common/a/y', 'common/b']),
+    (["common/a", "common/b"], ["common/a", "common/b", "old/a", "old/b"], [], ["common"]),
+    (
+        ["common/a/x", "common/a/y", "common/b/x", "common/b/y"],
+        ["common/a/x", "common/a/y", "common/b/x", "common/b/y"],
+        ["common/a/z"],
+        ["common/a/x", "common/a/y", "common/b"],
+    ),
 ]
 
+
 @pytest.mark.parametrize("paths1, paths2, excludes, expected", _test_reduce_common_paths_arg_values)
 def test_reduce_common_paths(paths1, paths2, excludes, expected):
     got = cdtb_export.reduce_common_paths(paths1, paths2, excludes)
     assert got == expected
 
+
+def test_wad_export_single_with_transform():
+    exporter = cdtb_export.Exporter("D:/export")
+    exporter.converters = [
+        # those are the converters used on raw
+        cdtb_export.ImageConverter([".dds", ".tga"]),
+        cdtb_export.BinConverter(extensions=[".bin"]),
+        cdtb_export.SKNConverter([".skn"]),
+    ]
+    exporter.add_path("D:/cdragon/Ahri.wad.client", "Ahri")
```

Comment:
Don't use hardcoded absolute paths in automatic/unitary tests.

Reply:
the test is removed from the updated version.